src/modules/cacheClient/cacheClient.service.ts

```diff
@@ -319,8 +319,10 @@ export class CacheClient implements ICacheClient {
      */
     private async isAuthenticated(): Promise<boolean> {
         try {
-            await this.httpClient.get<Asset[]>(`${TEST_LOGIN_ENDPOINT}${this._signerService.did}`);
-            return true;
+            const { data } = await this.httpClient.get<{
```

Comment:
I guess it should return user for whom access token was issued

Reply:
Hmm, IMO, a name that starts with "is" communicates that the return value is a boolean. Personally, I think the currently implementation is good.